modules/context/context.go

```diff
@@ -269,7 +269,7 @@ func (ctx *Context) ServerError(logMsg string, logErr error) {
 func (ctx *Context) serverErrorInternal(logMsg string, logErr error) {
 	if logErr != nil {
 		log.ErrorWithSkip(2, "%s: %v", logMsg, logErr)
-		if errors.Is(logErr, &net.OpError{}) {
+		if _, ok := logErr.(*net.OpError); ok || errors.Is(logErr, &net.OpError{}) {
```

Comment:
Would it be possible to add a test case for this?

Reply:
If I could have easily reproduced the problem I would have resolved this issue the first time and known that the original solution didn't work.